config/locales/en.yml

```diff
@@ -3522,23 +3522,28 @@ en:
           title: We need information about your health insurance status.
       nj_estimated_tax_payments:
         edit:
-          continue: If you did not already make any payments, click "Continue".
           description_html: |
-            <p>Please include the sum of the following:</p>
+            <p>Most people do <strong>not</strong> make these payments.</p>
+            <p>You are most likely to have made them if your employer <strong>doesn't</strong> withhold taxes in your paycheck or if you are self-employed.</p>
+            <p>These payments include:</p>
             <ul class="list--bulleted">
-              <li><a href="https://nj.gov/treasury/taxation/pdf/pubs/tgi-ee/git8.pdf" target="_blank" rel="noopener nofollow">Estimated income tax payments</a> you made throughout the year for your New Jersey state taxes
-                <ul class="list--bulleted">
-                  <li><strong>This does not include withholdings in your paycheck</strong></li>
-                </ul>
-              </li>
-              <li>Overpayments on your %{prior_year} return that you chose to apply to your %{filing_year} New Jersey taxes
-                <ul class="list--bulleted">
-                  <li>Note: you would have selected this option when filing your %{prior_year} return</li>
-                </ul>
-              </li>
+              <li><a href="https://www.nj.gov/treasury/taxation/njit20.shtml" target="_blank" rel="noopener nofollow">Estimated quarterly payments</a> that you made. These were due on April 15, June 16, and September 15 last year and January 15 this year. You would have paid these <a href="https://www1.state.nj.us/TYTR_RevTaxPortal/jsp/IndTaxLoginJsp.jsp" target="_blank" rel="noopener nofollow">online</a> or by mail.</li>
```

Comment:
[dust] I wonder if the `April 15, June 16, and September 15 last year and January 15 this year` dates are specific to this year or always the same? If they are likely to change maybe the string should have `2024` in the key to make it clear to revisit this string next year? but if it's low chance of change then nevermind